core/java/com/google/instrumentation/stats/ViewDescriptor.java

```diff
@@ -81,6 +79,43 @@ private ViewDescriptor(
   }
 
   /**
+   * Visitor for the ViewDescriptor type.
+   */
+  public abstract static class Visitor<T> {
+    public abstract T apply(DistributionViewDescriptor descriptor);
+
+    public abstract T apply(IntervalViewDescriptor descriptor);
+
+    /**
+     * Creates a Visitor with two {@link Function} objects.
```

Comment:
same comments. explaining the example is higher value than saying the vistor's params are functions :)

same also for createVisitor